### PR TITLE
feat: release for version 2.4.0 of the spec

### PR DESCRIPTION
**Pull request by smoya**
This PR is created upfront to reflect on a daily basis what things are included in the release.

Also, the reason to create this PR a long time before the release is to enable automation (bot keeps upstream branches always up to date with master, see this [action](https://github.com/asyncapi/spec/blob/master/.github/workflows/autoupdate.yml)) that we have in place to regularly update the release branch with whatever is changed in the master branch. So nobody has to do it manually.

### CODE REVIEW

**Review by derberg — APPROVED** [review]

**Comment by smoya** [comment]
@dalelane @fmvilas @derberg need a +1 on this + remove `do-not-merge` label so we can merge and release it 🙏

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@asyncapi/specs",
-  "version": "2.13.2",
+  "version": "2.14.0-2022-04-release.3",
```
Comment (magicmatatjahu) [inline package.json:3]: This should be probably changed to the `2.13.2`

Comment (derberg) [reply package.json:3]: no, RC should stay, `semantic-release` will figure it out, notice 2.13 and bump to 2.14 🍻

Comment (magicmatatjahu) [reply package.json:3]: Great ❤️

**Comment by smoya** [comment]
Only @dalelane approval is left and Ok to go!

**Pull request merged**